go/vt/vttablet/tabletmanager/vreplication/controller.go

```diff
@@ -95,6 +95,7 @@ func newController(ctx context.Context, params map[string]string, dbClientFactor
 	}
 	ct.id = int32(id)
 	ct.workflow = params["workflow"]
+	ct.lastWorkflowError = newLastError(fmt.Sprintf("VReplication controller %d for workflow %q", ct.id, ct.workflow), maxTimeToRetryError)
```

Comment:
Why are we doing this here? Can you please add a comment to clarify? I don't think it's related to the bug we're fixing is it? Did we just realize that each workflow's controller should have a non-nil newLastError struct (regardless of whether there's ever been an error or not?

Reply:
We were already initializing it while creating the controller. I was addressing your review comments on adding more detail to the name for better observability. I removed the previous init since it was superfluous.